packer/scripts/99-img_check.sh

```diff
@@ -359,9 +359,13 @@ function checkUpdates {
     elif [[ $OS == "CentOS Linux" ]]; then
         echo -en "\nChecking for available updates with yum, this may take a minute...\n\n"
         
-        update_count=$(yum list updates -q | grep -vc "Updated Packages")
+        update_count=$(yum list updates --security -q | grep -vc "Updated Packages")
```

Comment:
were you able to test this? maybe some sample output? just curious

Reply:
My system has all the relevant security updates, that being said the sample output mentions if there are any security updates available. 
`yum list updates --security
Last metadata expiration check: 0:04:22 ago on Wed 10 Feb 2021 02:45:37 PM UTC.
No security updates needed, but 29 updates available`